plugins/tiledb/io/TileDBWriter.cpp

```diff
@@ -526,16 +558,27 @@ void TileDBWriter::ready(pdal::BasePointTable &table)
 
 bool TileDBWriter::processOne(PointRef& point)
 {
+    m_array->close();
```

Comment:
I have lost track of the development of this writer, but it seems strange to do all this open/close of the array each time you want to write a point.